src/Bicep.Core/TypeSystem/Radius/V3/KnownComponents.cs

```diff
@@ -81,6 +81,68 @@ public static ComponentData MakeContainer()
+                    new TypeProperty("headers", headersType, TypePropertyFlags.None),
+                    new TypeProperty("kind", new StringLiteralType("httpGet"), TypePropertyFlags.Required),
+                    new TypeProperty("initialDelaySeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("failureThreshold", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("periodSeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                },
+                additionalPropertiesType: null,
+                additionalPropertiesFlags: TypePropertyFlags.None,
+                functions: null);
+
+            var tcp = new ObjectType(
+                name: "tcp",
+                validationFlags: TypeSymbolValidationFlags.Default,
+                properties: new TypeProperty[] {
+                    new TypeProperty("containerPort", LanguageConstants.Int, TypePropertyFlags.Required),
+                    new TypeProperty("kind", new StringLiteralType("tcp"), TypePropertyFlags.Required),
+                    new TypeProperty("initialDelaySeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("failureThreshold", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("periodSeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                },
+                additionalPropertiesType: null,
+                additionalPropertiesFlags: TypePropertyFlags.None,
+                functions: null);
+
+            var exec = new ObjectType(
+                name: "exec",
+                validationFlags: TypeSymbolValidationFlags.Default,
+                properties: new TypeProperty[] {
+                    new TypeProperty("command", LanguageConstants.String, TypePropertyFlags.Required),
+                    new TypeProperty("kind", new StringLiteralType("exec"), TypePropertyFlags.Required),
+                    new TypeProperty("initialDelaySeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("failureThreshold", LanguageConstants.Int, TypePropertyFlags.None),
+                    new TypeProperty("periodSeconds", LanguageConstants.Int, TypePropertyFlags.None),
+                },
+                additionalPropertiesType: null,
+                additionalPropertiesFlags: TypePropertyFlags.None,
+                functions: null);
+
+            var healthProbeType = new DiscriminatedObjectType(
+                name: "healthProbe",
+                validationFlags: TypeSymbolValidationFlags.Default,
+                discriminatorKey: "kind",
+                unionMembers: new ITypeReference[]{httpGet, tcp, exec}
+                );
+            
+            var readinessProperty = new TypeProperty("readiness", healthProbeType, TypePropertyFlags.None, "Readiness check");
```

Comment:
In the schema these are different:

- readinessProbe
- livenessProbe 

https://github.com/Azure/radius/blob/main/pkg/radrp/schemav3/components/container-component.json#L58